libenkf/src/res_config.c

```diff
@@ -81,6 +101,44 @@ analysis_config_type * res_config_get_analysis_config(
   return res_config->analysis_config;
 }
 
+ert_workflow_list_type * res_config_get_workflow_list(
+                    const res_config_type * res_config
+        ) {
+  return res_config->workflow_list;
+}
+
+subst_list_type * res_config_get_subst_list(
+                    const res_config_type * res_config
+                    ) {
+  return subst_config_get_subst_list(res_config->subst_config);
+}
+
+subst_func_pool_type * res_config_get_subst_func_pool(
+                    const res_config_type * res_config
+                    ) {
+  return subst_config_get_subst_func_pool(res_config->subst_config);
+}
+
+subst_config_type * res_config_get_subst_config(
+                    const res_config_type * res_config
+                   ) {
+  return res_config->subst_config;
+}
+
+char * res_config_alloc_working_directory(const char * user_config_file) {
```

Comment:
This should be internalized creation time as a member variable, and we should have `res_config_get_working_directory( )`